catalyst_voices/uikit_example/lib/main.dart

```diff
@@ -27,6 +28,7 @@ class _UIKitExampleAppState extends State<UIKitExampleApp> {
     return MultiBlocProvider(
       providers: [
         BlocProvider.value(value: _sessionBloc),
+        BlocProvider.value(value: _userProfileBloc),
```

Comment:
Should we add it in Voices `App`->`MultiBlocProvider`? As well as register in `Dependencies`

Reply:
Will do. Wasn't sure when we will move to work on cat-voices app instead of UI kit

Reply:
Soon, after my PR i think :D

Reply:
Ok. Is there anything that is blocking this PR from merging then? Once you're finished I will do the changes